modules/auxiliary/server/relay/smb_to_ldap.rb

```diff
@@ -0,0 +1,118 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::SMB::RelayServer
```

Comment:
I found a tiny bug in our `HashCapture` module which gets included by `RelayServer`. Unrelated to this PR but figured I'd reference as I found it while testing: https://github.com/rapid7/metasploit-framework/pull/19842/files

Reply:
Good catch! Thank you for letting me know.